dev.deedles.Trayscale.yml

```diff
@@ -0,0 +1,49 @@
+app-id: dev.deedles.Trayscale
+
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: trayscale
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --filesystem=/run/tailscale:ro
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Wouldn't it make more sense to bundle tailscale CLI with the app?

Reply:
That's an option. I _had_ looked into bundling the whole of Tailscale, including `tailscaled`, with the app, but that didn't work because it needed root permissions to create a bridge network device. It might be possible to bundle just the `tailscale` binary, but it might require the version to match between `tailscaled` and `tailscale`, in which case that'll make usage rather awkward. I'll try bundling it with a version purposefully mismatched from my system's and see what happens.

Reply:
Bundling it worked, though it does add architecture-specific sources. I was able to remove the `--talk-name=org.freedesktop.Flatpak` permission because of it, though.